src/NuGet.Core/NuGet.PackageManagement/Strings.resx

```diff
@@ -521,4 +521,13 @@
 2 - Package ID
 3 - Package Sources</comment>
   </data>
-</root>
+  <data name="PackageNamespaceMatchFound" xml:space="preserve">
+    <value>Package namespace matches found for package id '{0}' are: '{1}'</value>
```

Comment:
We are already inconsistent in resource strings, but I'd lean towards `ID` over `id`.  This feedback applies to the next resource string too.